java-extension/com.microsoft.java.test.plugin/target.target

```diff
@@ -24,4 +29,4 @@
             <repository location="http://download.eclipse.org/tools/orbit/R-builds/R20170516192513/repository"/>
```

Comment:
add `<unit id="org.apache.commons.lang3" version="3.1.0.v201403281430"/>` here to acquire